rulebook.md

```diff
@@ -596,6 +592,15 @@ Se considera la combinación de dos espadas cortas como una combinación especí
 - Un jugador que use esta combinación de armas puede atacar con la otra espada si se desprende de una de manera por el motivo que fuera. Podrá recoger su espada en cualquier momento del punto o la finalizar el mismo.
 - Un jugador que use esta combinación de armas podrá pinear a un jugador con cada una de sus espadas.
 
+Si esta acción se considera que se ejecuta con fuerza excesiva o brutalidad, deberá ser sancionada con una falta técnica.
+
+## 5.17 JUEGO IMPRUDENTE
+Este término se refiere a las acciones del juego en las que un jugador adopta una postura negligente, arriesgada o temeraria y sin tener en cuenta las consecuencias pudiendo poner en peligro la seguridad de los demás jugadores.
+
+Si el jugador que recibió la acción considera que la acción no ha sido imprudente, prevalecerá el acuerdo entre jugadores.
```

Comment:
Ojo, mi opinión solo, pero yo aquí no dejaría al jugador invalidar una acción del equipo arbitral por acuerdo entre jugadores. Eso debería ser limitado a lances de juego (tuya, mía, dobles) y no ser válido en situaciones en las que esté en riesgo la integridad física de alguien (pueden darme una toña como un piano, que por suerte no me pase nada y me lo tome de risas, y eso no debería impedir que la persona que realizó la acción sea sancionada por su temeridad y posibles consecuencias

Reply:
Esta es una cuestión muy interesante